orchagent/portsorch.cpp

```diff
@@ -5314,6 +5351,20 @@ bool PortsOrch::initializePort(Port &port)
         port.m_oper_status = SAI_PORT_OPER_STATUS_DOWN;
     }
 
+    // initalize port flap count
+    if (!flapCount.empty())
+    {
+        try
+        {
+            port.m_flap_count = to_uint<std::uint32_t>(flapCount);
+        }
+        catch (const std::exception &e)
+        {
+            SWSS_LOG_ERROR("Failed to get port (%s) flap_count: %s", port.m_alias.c_str(), e.what());
```

Comment:
This is going to throw an unwanted error during WB from old versions which doesn't have this field in DB. Please change to DEBUG as this is not a real error scenario

Reply:
@prsunny if the value is not there in the DB, flapCount default value = "0" (see line 5317 above)